middlewares/validators/search.validator.js

```diff
@@ -3,7 +3,7 @@ const VALIDATOR = require("./validator.helper");
 
 module.exports = {
     searchQueryValidator: [
-        VALIDATOR.alphaValidator("param", "model", false),
+        VALIDATOR.searchModelValidator("param", "model", false),
```

Comment:
Just a nitpick that here we take 3 inputs, but the actuall function searchModelValidator only takes a fieldLocation and fieldName. Would prefer we either remove the 'false' argument, or change searchModelValidator to be inline with other validators, and take a boolean for optional.

Reply:
Good catch